handlers.go

```diff
@@ -85,6 +85,12 @@ func (r *oauthProxy) oauthAuthorizationHandler(w http.ResponseWriter, req *http.
 	}
 
 	authURL := client.AuthCodeURL(req.URL.Query().Get("state"), accessType, "")
+	// idp_hint config
+	idpHint := r.config.IdpHint
+	if idpHint != "" {
+		authURL += "&kc_idp_hint=" + idpHint
```

Comment:
Could there sometimes be other url parameters already in authURL? I am thinking "&" vs "?"
If that is never a possibility please disregard, otherwise I suggest adding some pre-inspection of authURL before appending.